add-on/panels/js/panel.jsx

```diff
@@ -128,16 +128,16 @@ loop.panel = (function(_, mozL10n) {
       var locale = mozL10n.getLanguage();
       var tosHTML = mozL10n.get("legal_text_and_links3", {
         "clientShortname": mozL10n.get("clientShortname2"),
-        "terms_of_use": React.renderToStaticMarkup(
+        "terms_of_use": "<a>Fake</a>", /* ReactDOM.render(
```

Comment:
Scrub the previous comment about upgrading L10n.js. 

There's a simpler way. This section change be changed to:

      var tosString =
        '<a href="' + this.state.terms_of_use_url + '" target="_blank">' +
        mozL10n.get("legal_text_tos") +
        "</a>";

      var privacyString =
        '<a href="' + this.state.privacy_notice_url + '" target="_blank">' +
        mozL10n.get("legal_text_privacy") +
        "</a>";

      var tosHTML = mozL10n.get("legal_text_and_links3", {
        "clientShortname": mozL10n.get("clientShortname2"),
        "terms_of_use": tosString,
        "privacy_notice": privacyString
      });

It functions the same, just doesn't take advantage of the static markup rending that react has. However, I think that's reasonable to loose considering the react changes.

There's also a similar ToS link setup in the standalone that can be changed in the same manner.